worker/utils/logger.js

```diff
@@ -1,19 +1,39 @@
-const workerUtils = require("./utils");
+const workerUtils = require('./utils');
 
 class LoggerClass {
-  // pass in a job payload to setup class
-  constructor(currentJob) {
-    this.currentJob = currentJob;
-  }
+    // pass in a job payload to setup class
+    constructor(currentJob) {
+        this.currentJob = currentJob;
+    }
 
-  sendSlackMsg(message) {
-    workerUtils.populateCommunicationMessageInMongo(this.currentJob, message);
-  }
+    sendSlackMsg(message) {
+        workerUtils.populateCommunicationMessageInMongo(this.currentJob, message);
+    }
 
-  save(message) {
-    workerUtils.logInMongo(this.currentJob, message);
-  }
+    save(message) {
+        workerUtils.logInMongo(this.currentJob, message);
+    }
+
+    static trimOutputForUserFacingLogMessages(output) {
+        let trimmedOutput = '';
+        const splitOutput = output.split('\n');
+        splitOutput.forEach((line) => {
+            if (line.indexOf('WARNING') > -1 || line.indexOf('ERROR') > -1 || line.indexOf('INFO') > -1) {
```

Comment:
Having an INFO message probably doesn't require that we dump the entire build log, but the point is a little moot because we don't currently *have* any INFO messages :D

Reply:
It would only publish the INFO log line -- and I did see one in testing. It might have been from a legacy build.

Reply:
Correction - it's next gen: the snooty parser version is an INFO log.